api/models.py

```diff
@@ -21,6 +21,9 @@ class UserProfile(models.Model):
 
 	#Relations
 	user = models.OneToOneField(User) 
+	def __unicode__(self):
+		return u'%s' % (self.user.username)	
```

Comment:
You're awesome for adding the `__unicode__` method! :+1: 

Although, you could simplify this line by writing just `return self.user.username`(same for the return under the Projects model). Using string replacements (the `%s` part) makes sense if you have more variables, but not when you have just one.